src/go/configgenerator/route_generator.go

```diff
@@ -32,8 +32,10 @@ import (
 )
 
 const (
-	routeName       = "local_route"
-	virtualHostName = "backend"
+	routeName               = "local_route"
+	redirectRouteName       = "redirect_route"
+	virtualHostName         = "backend"
+	redirectVirtualHostName = "redirect_backend"
```

Comment:
If MakeRedirectRouteConfig is moved, ditto.

Reply:
Done